src/Options/ParseOptions.hpp

```diff
@@ -39,15 +39,15 @@
 // include this file.)
 
 inline Option::Option(YAML::Node node, OptionContext context) noexcept
-  // clang-tidy: YAML::Node not movable (as of yaml-cpp-0.5.3)
```

Comment:
Please try to avoid formatting the whole file and only format the lines you edited. `git clang-format` will do this for you.

Reply:
I changed my mind. Please remove the white space changes and then I'll look. I don't really want to stare at code trying to figure out if it actually changed and then figure out what changed and why.

Reply:
The clang-format is in a separate commit, clang-formatting the option files. Just look at the other commit.